### PR TITLE
Docs for self-signed CA workarounds

### PR DESCRIPTION
**Pull request by dayleparker**
Hi Suyog,

I've added an appendix on adding a self-signed CA, and notes in OpenStack and SCVMM sections linking to this appendix, as they seem to be the only providers affected by this now (ie. there is SSL validation in the UI, but no option for a self-signed cert). 
Please let me know if you have questions or comments -- I'll apply any SME feedback later so we can get this in for GA.

https://bugzilla.redhat.com/show_bug.cgi?id=1448711

http://file.bne.redhat.com/~dayparke/CloudForms/SSLCAcert/build/tmp/en-US/html-single/#app-self_signed_CA

Many thanks!
Dayle

### CODE REVIEW

**Comment by suyogsainkar** [comment]
@dayleparker Hi Dayle, I've completed a peer review of the updates and it looks all good to me. As discussed, I'm merging it to master to get it in for the 4.5 GA. I have the following two optional suggestions for later -

1. For 1.3.1. Adding an OpenStack Infrastructure Provider: I was wondering if we should combine the existing back-to-back Note boxes into one with bullets? Example:

[NOTE]
*  You can set whether Red Hat CloudForms should use the Telemetry service or Advanced Message Queueing Protocol (AMQP) for event monitoring. If you choose Telemetry, you should first configure the ceilometer service on the undercloud to store events. See Section 1.3.1.1, “Configuring the Undercloud to Store Events” for instructions.  For more information, see OpenStack Telemetry (ceilometer) in the Red Hat OpenStack Platform Architecture Guide.

* To authenticate the provider using a self-signed Certificate Authority (CA), configure the CloudForms appliance to trust the certificate using the steps in Section A.1, “Using a Self-Signed CA Certificate” before adding the provider.  

2.  For the note box under appendix A.1. Using a Self-Signed CA Certificate, would rewording the text something along the following make it more straightforward for customers? 

[NOTE]
"This procedure is not required for OpenShift Container Platform, Red Hat Virtualization, or middleware manager providers as they have the option to select **SSL trusting custom CA** as a **Security Protocol** in the user interface. These steps are needed only for providers that do not have the option to configure self-signed CA's in the user interface." 

Please let me know what you think. Thanks for all the updates, Dayle! 

Cheers, Suyog

**Comment by dayleparker** [comment]
Hi @suyogsainkar, 
Thanks for your feedback and review! Good ideas -- I've implemented them in a new PR: https://github.com/ManageIQ/manageiq_docs/pull/380.
Cheers,
Dayle
